### ISSUE TITLE
little tips for this repo

### ISSUE THREAD

**Issue by cvtower** [xxradon/IGCV3-pytorch#3]
Hi @xxradon ,

Very nice work!
Here i have 2 tips for this repo which will help to reproduce performance mentioned in the paper:
1. change lr_schedule(mbv2&&igcv3). *0.1 for epoch 200/300/350;
2. igcv3 build block should remove the first relu.
Then you will obtain wanted accuracy.

**Comment by thinkInJava33** [comment]
@cvtower  i follow your suggustions,but i still can not reproduce the result on cifar100.

**Comment by cvtower** [comment]
hi @thinkInJava33 , I reproduced the performance from paper last year, and I will upload my modified version and trained models as well as tensorboard records later to my private repo-seesawnet_pytorch.

**Comment by thinkInJava33** [comment]
@cvtower  thank you,i am wating for it,and i hope it won't be too long.

**Comment by cvtower** [comment]
> @cvtower thank you,i am wating for it,and i hope it won't be too long.

hi @thinkInJava33 ,
https://github.com/cvtower/SeesawNet_pytorch/tree/master/cifar_test/78.21_igcv3_g2g2
